LayoutTests/editing/selection/setting-selection-does-not-focus-unless-selected.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Setting selection through APIs does not focus unless selection is inside the text control.</title>
```

Comment:
It would be better this was in the body so that the output will contain this string.s